index.html

```diff
@@ -97,7 +97,7 @@ <h1>CodeMeta generator v3.0</h1>
 
                 <br />
                 <span class="field-description" id="identifier_descr">
-                    such as ISBNs, GTIN codes, UUIDs, etc. see <a href="http://schema.org/identifier">http://schema.org/identifier</a>
+                    such as ISBNs, GTIN codes, UUIDs, etc. see <a href="https://schema.org/identifier">schema.org/identifier</a>
```

Comment:
```suggestion
                    such as ISBNs, GTIN codes, UUIDs, etc. see <a href="https://schema.org/identifier">https://schema.org/identifier</a>
```

I'd say the scheme matters here because `https://schema.org/identifier` is also the term itself.